code/src/index.css

```diff
@@ -1,13 +1,296 @@
+/* COLORs 
+
+FCF9EA - light beige
+BADFDB ice blue 
+F8A978 light terracotta
+FFC5A1 lightest peach  
+
+*/
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: rgb(88, 80, 80); 
+  background-image: url("../public/images/yarn.jpg"); 
+  background-repeat: no-repeat;
+  background-size: 1800px;
```

Comment:
A small thing i noticed is that i can scroll to the right, so the background is bigger than my screen (also in mobile mode)